src/mlnet/ProgressBar/ProgressBarBase.cs

```diff
@@ -16,20 +16,22 @@ static ProgressBarBase()
             Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
         }
 
-        protected readonly DateTime _startDate = DateTime.Now;
+        public readonly DateTime StartDate;
```

Comment:
not a big deal either way but I don't see it -- why this had to switch to pubic?

Reply:
Apparently, there was one usage outside the ProgressBarBase inheritance chain